### PR TITLE
Fixed sync icon in dataset view title

### LINKED ISSUE #10035: Sync table Layout seems broken on the title
screen shot 2016-10-05 at 13 06 10

When importing from https://data.cityofnewyork.us/api/geospatial/e2f7-cs7i?method=export&format=GeoJSON

### PR DESCRIPTION
**Pull request by MariaCheca**
Fixes #10035

screen shot 2016-10-05 at 16 28 54

CR @xavijam

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/dataset/dataset-header/dataset-header.tpl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="Editor-HeaderInfo-inner">
   <div class="Editor-HeaderInfo-title u-bSpace--m u-flex u-alignCenter">
     <% if (isSync && isOwner) { %>
-      <div class="Editor-HeaderInfo-details">
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/dataset/dataset-header/dataset-header.tpl]: So *-details class is not needed anymore?

Comment (MariaCheca) [inline lib/assets/javascripts/cartodb3/dataset/dataset-header/dataset-header.tpl]: Yes it is! It's used some lines below: https://github.com/CartoDB/cartodb/pull/10046/files#diff-95c7db2bb052044bfe92249d74de60d8L13

**Comment by MariaCheca** [comment]
I fixed an extra margin in the dropdown button (three-dots icon), cc @piensaenpixel

**Comment by xavijam** [comment]
Fine by me, @piensaenpixel ?

**Comment by piensaenpixel** [comment]
+1, i'd like take a look these componentes, but ... we'll do

**Comment by xavijam** [comment]
Mere @MariaCheca  and deploy please 💋

**Comment by MariaCheca** [comment]
🚢 🇮🇹

**Pull request merged**